src/main/java/com/bandwidth/controller/CallbacksController.java

```diff
@@ -1,92 +1,102 @@
-import com.bandwidth.Environment;
-import com.bandwidth.Model.VoiceCallback;
-import com.bandwidth.exceptions.ApiException;
-import com.bandwidth.http.response.ApiResponse;
-import com.bandwidth.voice.bxml.verbs.*;
-import com.bandwidth.voice.controllers.APIController;
-import org.apache.commons.io.FileUtils;
+import com.bandwidth.Main;
+import com.bandwidth.sdk.ApiClient;
+import com.bandwidth.sdk.ApiResponse;
+import com.bandwidth.sdk.ApiException;
+import com.bandwidth.sdk.auth.HttpBasicAuth;
+import com.bandwidth.sdk.Configuration;
+
+import com.bandwidth.sdk.model.InitiateCallback;
+import com.bandwidth.sdk.model.RecordingAvailableCallback;
+import com.bandwidth.sdk.api.RecordingsApi;
+import com.bandwidth.sdk.model.bxml.*;
+import com.bandwidth.sdk.model.bxml.Record;
+import jakarta.xml.bind.JAXBContext;
+import jakarta.xml.bind.JAXBException;
+
+import java.io.IOException;
+import java.io.File;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-import javax.servlet.http.HttpServletRequest;
-import java.io.File;
-import java.io.IOException;
-import java.io.InputStream;
-import java.net.MalformedURLException;
-
 @RestController
 @RequestMapping("callbacks")
 public class CallbacksController {
-
     Logger logger = LoggerFactory.getLogger(CallbacksController.class);
 
-    private String username = System.getenv("BW_USERNAME");
-    private String password = System.getenv("BW_PASSWORD");
-    private String accountId = System.getenv("BW_ACCOUNT_ID");
-    private String applicationId = System.getenv("BW_VOICE_APPLICATION_ID");
-
-    private BandwidthClient client = new BandwidthClient.Builder()
-            .voiceBasicAuthCredentials(username, password)
-            .environment(Environment.PRODUCTION)
-            .build();
+    public final String username = System.getenv("BW_USERNAME");
+    public final String password = System.getenv("BW_PASSWORD");
+    public final String accountId = System.getenv("BW_ACCOUNT_ID");
+    public final String baseUrl = System.getenv("BW_BASE_URL");
 
-    private APIController controller = client.getVoiceClient().getAPIController();
+    public ApiClient defaultClient = Configuration.getDefaultApiClient();
+    public HttpBasicAuth Basic = (HttpBasicAuth) defaultClient.getAuthentication("Basic");
 
-    @PostMapping("/callInitiatedCallback")
-    public String voiceCallback(@RequestBody VoiceCallback callback, HttpServletRequest request) throws IOException, MalformedURLException {
+    public final RecordingsApi recordingsApi = new RecordingsApi(defaultClient);
 
-        logger.info(request.getServletPath() + " requested");
+    @PostMapping("/inbound")
```

Comment:
we should leave this as `callInitiatedCallback` for consistency with the other samples